cpp/ast.py

```diff
@@ -1428,6 +1428,7 @@ def _get_bases(self):
     def _get_class(self, class_type, templated_types):
         class_name = None
         class_token = self._get_next_token()
+        name_tokens = []
```

Comment:
Hmm, I'm not seeing it. So `assert_parse` fails, but how does that relate to `name_tokens`?

Perhaps an example C++ file would help me understand the problem better.